README.md

```diff
@@ -58,6 +58,7 @@ journalctl -fu docker.service # Journalctl/Systemd
  and log location. Example:
 ```
  {"MaxLogAgeDays": 28,
+ "MaxLogFiles": 3,
  "MaxLogSizeMb": 100,
```

Comment:
Not sure why this is 100? Isn't it cumbersome to load 100mb files while debugging? I suggest changing this to 10.

Reply:
as explained above.

Reply:
I had 10 x 100MB files and those filled up the root FS, most distros for docker are pretty small and 1GB or even 500MB of logs is going to be impossible to host. We need small log files and few of those.

Reply:
Okay sure, I will set the sample values as shown below.

```
"MaxLogFiles": 3,
  "MaxLogSizeMb": 10,
```

At any given time, there will be ~30MB size will be used mostly. Please share your comment to close on this one. I will update the diff then after.